crates/optimism/payload/src/builder.rs

```diff
@@ -785,7 +786,7 @@ impl<EvmConfig: ConfigureEvmEnv, N: NodePrimitives> OpPayloadBuilderCtx<EvmConfi
 impl<EvmConfig, N> OpPayloadBuilderCtx<EvmConfig, N>
 where
     EvmConfig: ConfigureEvmFor<N>,
-    N: OpPayloadPrimitives,
+    N: OpPayloadPrimitives<_TX = OpTransactionSigned>,
```

Comment:
for example, can we make SignedTx = OpTransactionSigned? Any reason this needs to be generic in OpPayloadPrimitives